tests/tests.cpp

```diff
@@ -49,6 +49,160 @@ TEST(Serialize64, serialize) {
     ASSERT_EQ(in, out);
 }
 
+TEST(PayCrypterTest, allFeatures)
```

Comment:
Would be good if this could be moved to its own test file, not in the generic `test.cpp` (maybe `paycrypt_test.cpp` or so)